tofu/geom/_core.py

```diff
@@ -4286,15 +4286,16 @@ def get_sample(self, res=None, resMode='abs', DL=None, method='sum', ind=None,
 
         # Launch    # NB : find a way to exclude cases with DL[0,:]>=DL[1,:] !!
         # Todo : reverse in _GG : make compact default for faster computation !
-        k, reseff, ind = _GG.LOS_get_sample(Ds, us, res, DL,
+        nlos = Ds.shape[1]
+        k, reseff, lind = _GG.LOS_get_sample(nlos, res, DL,
```

Comment:
I saw the variable name `ind` was being used twice ! renamed it to `lind` for that reason